snapcraft/plugins/catkin.py

```diff
@@ -105,6 +105,7 @@ def env(self, root):
                 root,
                 self.options.rosversion),
             'ROS_MASTER_URI=http://localhost:11311',
```

Comment:
Heh, I don't quite understand what you're trying to draw my attention to. That you'd like that variable removed eventually?